src/test/java/net/openhft/chronicle/network/UberHandlerTest.java

```diff
@@ -289,7 +290,7 @@ public void onRead(@NotNull WireIn inWire, @NotNull WireOut outWire) {
 
         @Override
         public void onInitialize(WireOut outWire) throws RejectedExecutionException {
-            throw new RejectedExecutionException("Rejected in onInitialize");
+            throw new RejectedHandlerException("Rejected in onInitialize");
```

Comment:
This was a typo from previous commit

Reply:
That I only noticed now because Sonar is working :tada:

Reply:
hehe nice